plugins/org.wso2.developerstudio.eclipse.esb.libraries/.classpath

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/synapse-core_2.1.7.wso2v83.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/synapse-core_2.1.7.wso2v87.jar"/>
```

Comment:
Shall we use the released synapse-core version?

Reply:
Added new synapse-core version